superdesk/vocabularies/__init__.py

```diff
@@ -24,3 +26,5 @@ def init_app(app):
         'category': 'cvs',
         'label': 'Prefered CV items',
     })
+
+    item_published.connect(add_missing_keywords)
```

Comment:
@petrjasek Can we please have a system setting for this (we want our keywords vocabulary to be managed by admins).